material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -660,6 +660,9 @@ input.md-search__input::placeholder {
   }
 
   .md-sidebar__scrollwrap {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
```

Comment:
```suggestion
```

This isn't necessary and it actually changes the behavior of the left sidebar navigation